src/Types.tsx

```diff
@@ -1,9 +1,10 @@
 import { Point } from "react-simple-maps";
 
 export type rowProp = string | Point | number;
```

Comment:
Let's capitalize types.

Reply:
Where else would camelCase be acceptable in naming, besides functions?